filesystem/drive.go

```diff
@@ -21,8 +24,38 @@ func DoesDriveExistWithTimeout(driveName string, maxAttempts int) bool {
 
 //DoesDriveExist does a single stat call to check if a drive exists
 func DoesDriveExist(driveName string) bool {
-	if _, err := execute.Command("stat", []string{driveName}, ""); err != nil {
+	if _, err := os.Stat(driveName); os.IsNotExist(err) {
```

Comment:
This is nice - it's always better to use language builtins than shell out to `stat` - not sure why I made that decision.